src/frontend/packages/lib_markdown/package.json

```diff
@@ -39,6 +39,7 @@
     "@codemirror/state": "*",
     "@codemirror/view": "*",
     "@mdx-js/mdx2": "*",
+    "@openfun/cunningham-react": "*",
```

Comment:
Same comment as above, you should not have it in both `dependencies` and `devDependencies`